pom.xml

```diff
@@ -328,9 +304,65 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <groupId>org.jacoco</groupId>
+                    <artifactId>jacoco-maven-plugin</artifactId>
+                    <version>0.8.0</version>
```

Comment:
Move these versions to properties, similar to the other plugins

Reply:
Done.